addons/l10n_pe/demo/l10n_pe_demo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="peru_partner" model="res.partner">
+        <field name="name">GRUPO COMERCIALIZADOR DIGITAL SOCIEDAD ANONIMA CERRADA - GRUPO COMERCIALIZADOR DIGITAL S.A.C.</field>
```

Comment:
It does not make sense to give it another name as it will be overwritten by the company, but it would be good to have a demo partner.  (think about demoing edi as well)